cli/command/trust/sign.go

```diff
@@ -18,19 +18,26 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type signOptions struct {
+	local bool
+}
+
 func newSignCommand(dockerCli command.Cli) *cobra.Command {
+	options := signOptions{}
 	cmd := &cobra.Command{
 		Use:   "sign IMAGE:TAG",
 		Short: "Sign an image",
 		Args:  cli.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runSignImage(dockerCli, args[0])
+			return signImage(dockerCli, args[0], options)
```

Comment:
Small thing, but would you mind making the `args[0]` part of options struct by adding an `imageName` field? We do this for most commands.